sdks/python/examples/cancellation/worker.py

```diff
@@ -1,30 +1,26 @@
 import asyncio
 
-from hatchet_sdk import BaseWorkflow, Context, Hatchet
+from hatchet_sdk import Context, EmptyModel, Hatchet
 
 hatchet = Hatchet(debug=True)
 
-wf = hatchet.declare_workflow(on_events=["user:create"])
+wf = hatchet.workflow(name="CancelWorkflow", on_events=["user:create"])
```

Comment:
nit: maybe its time to drop the default on_events from all the examples

Reply:
we'll need to swap tests to .run

Reply:
yeah will do, was thinking about this too 👍 it should make running different examples simpler too b/c there won't be a bunch of phantom runs queued for workflows that aren't being registered anymore which has been annoying in the current setup